### PR TITLE
Aag 1591 callback logs

### PR DESCRIPTION
**Pull request by natashashafqat**
Adding debug logs for:
- adLoaded
- adShown
- adClicked
- adEnded
- adClosed

Debug logs can now be seen in Logcat, and can easily be found by keeping a filter on for `Event callback: `.

Video below showing debug logs being registered and easily seen for the above callbacks.

https://user-images.githubusercontent.com/48458132/152332378-b6dee0f5-41da-4c4f-b136-2be1805b9419.mov

### CODE REVIEW

File: superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SABannerAd.java [hunk 1]
```diff
@@ -162,7 +162,9 @@ public void load (final int placementId) {
                     canPlay = response.isValid();
                     setAd(response.isValid() ? response.ads.get(0) : null);
                     if (listener != null) {
-                        listener.onEvent(placementId, response.isValid() ? SAEvent.adLoaded : SAEvent.adEmpty);
+                        SAEvent eventToSend = response.isValid() ? SAEvent.adLoaded : SAEvent.adEmpty;
+                        listener.onEvent(placementId, eventToSend);
+                        Log.d("SABannerAd.load.PL_ID", "Event callback: " + eventToSend.toString());
```
Comment (gunhansancar) [inline superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SABannerAd.java:167]: Log tags (the first parameter) should be same for everything, or at least the file based, and it should be short. Please let's use the same tag, it is better for filtering as well.

File: superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SABannerAd.java [hunk 2]
```diff
@@ -228,7 +230,9 @@ public void load (final int placementId, final int lineItemId, final int creativ
                     canPlay = response.isValid();
                     setAd(response.isValid() ? response.ads.get(0) : null);
                     if (listener != null) {
-                        listener.onEvent(placementId, response.isValid() ? SAEvent.adLoaded : SAEvent.adEmpty);
+                        SAEvent eventToSend = response.isValid() ? SAEvent.adLoaded : SAEvent.adEmpty;
+                        listener.onEvent(placementId, eventToSend);
+                        Log.d("SABannerAd.load.ALL_IDs", "Event callback: " + eventToSend.toString());
```
Comment (gunhansancar) [inline superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SABannerAd.java:235]: Same here, I wish we started using the refactored code which contains better logger already

File: superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SAInterstitialAd.java [hunk 3]
```diff
@@ -187,6 +188,7 @@ private void close () {
      * @param placementId   the Ad placement id to load data for
      * @param context       the current context
      */
+    @SuppressLint("LongLogTag")
```
Comment (gunhansancar) [inline superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SAInterstitialAd.java:191]: Let's not suppress warnings and solve the issue by using the smaller same tags

**Review by gunhansancar — CHANGES_REQUESTED** [review]
Please consider my comments for all the changes as I didn't mark them all

File: superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SABannerAd.java (hunk 1 above)
Comment (natashashafqat) [reply superawesome-base/src/main/java/tv/superawesome/sdk/publisher/SABannerAd.java:167]: Makes sense 👍  I wasn't sure on how to label the tags. I can just do file name? So in this case, just `SABannerAd`?

**Review by gunhansancar — APPROVED** [review]

**Pull request merged**